src/__stories__/chip-story.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import {Chip, IconLightningFilled, useTheme} from '..';
+import {StorySection} from './helpers';
+
+export default {
+    title: 'Components/Others/Chip',
+};
+
+export const Default: StoryComponent = () => {
+    const {colors} = useTheme();
+    return (
+        <div data-testid="chip-story">
+            <StorySection title="Default">
+                <Chip>Chip</Chip>
+            </StorySection>
+            <StorySection title="Closeable">
+                <Chip
+                    onClose={() => {
+                        window.alert('closed');
+                    }}
+                >
+                    Chip closeable
+                </Chip>
+            </StorySection>
+            <StorySection title="With icon">
+                <Chip icon={<IconLightningFilled size={16} color={colors.neutralMedium} />}>
```

Comment:
Different usage between chips and future tags

would be nice to have same @pladaria 
https://mistica-web-g8xmuzffg-tuentisre.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AVAhgcwAQBcCeADjALwA6IBAThALYTlYCSkAdscMxCwMo5qUAxAJYAbETCgBfAHykWWLAAVqdOUgD06DLJYhJQA

Reply:
But this is not the same situation. In Tags the icon properties (size, color) are managed by the tag component.

Chip component doesn't specify any color so the icon prop is free

Reply:
well, it does. The icon size must be 16